wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -145,18 +145,31 @@ pub enum WalletCommand {
         transaction: HexEncoded<SignedTransaction>,
     },
 
+    /// Abandon an unconfirmed transaction, and make the consumed inputs available to be used again
+    AbandonTransaction {
```

Comment:
Shouldn't there also be a command to list all the transactions (or their ids) that can be abandoned, I.e., pending? Something like `ListPendingTransactions` is probably needed.

Reply:
Renamed ListAbandonableTransactions to ListPendingTransactions